### DISCUSSION TITLE
Hashtags (#) in query

### DISCUSSION THREAD

**Discussion by FerusAndBeyond** [PostgREST/postgrest#2334] [Q&A] [answered]
When I try to add a hashtag in a string equality comparison, for example "...?hello=eq.some#string", I can't get it to work. I tried URI-encode it (%23) and when logging I see it has been translated serverside to `%2523`. But it doesn't return what is supposed to be returned.

* PostgreSQL version: 10.5
* PostgREST version: v9.0.1

I can't find anything in the documentation about this.

**Comment by steve-chavez** [comment, marked as answer]
How are you making the request??

**Reply by laurenceisla** [reply]
Just to add, I tested v9.0.1 with PG14 and a `curl` works OK with url encoding:

```bash
$ curl "http://localhost:3000/users?email=eq.Test%23"
> [{"user_id":1,"name":"User","email":"Test#"}]
```

**Reply by FerusAndBeyond** [reply]
I was making the request with the angular httpclient (https://angular.io/api/common/http/HttpClient), but now when I tested it with Python it worked, thus, I realized that it is this library that is causing the problem.
